### PR TITLE
Use OIDC logout url if it is provided.

### PR DESCRIPTION
**Pull request by mythmon**

### CODE REVIEW

File: recipe-server/client/control_new/components/common/CurrentUserDetails.js [hunk 1]
```diff
@@ -3,21 +3,23 @@ import PropTypes from 'prop-types';
...
 )
 export default class CurrentUserDetails extends React.Component {
   static propTypes = {
     user: PropTypes.object.isRequired,
+    logoutUrl: PropTypes.string.isRequired,
   }
 
   render() {
-    const { user } = this.props;
+    const { user, logoutUrl } = this.props;
```
Comment (rehandalal) [inline recipe-server/client/control_new/components/common/CurrentUserDetails.js]: nit: I've been trying to keep these alphabetized

File: recipe-server/client/control_new/components/common/CurrentUserDetails.js [hunk 2]
```diff
@@ -3,21 +3,23 @@ import PropTypes from 'prop-types';
...
   state => ({
     user: getCurrentUser(state),
-  }),
+    logoutUrl: getLogoutUrl(state),
+  })
 )
 export default class CurrentUserDetails extends React.Component {
   static propTypes = {
     user: PropTypes.object.isRequired,
+    logoutUrl: PropTypes.string.isRequired,
```
Comment (rehandalal) [inline recipe-server/client/control_new/components/common/CurrentUserDetails.js]: nit: I've been trying to keep these alphabetized

File: recipe-server/normandy/base/api/views.py [hunk 3]
```diff
@@ -64,6 +64,7 @@ def get(self, request):
         return Response(ServiceInfoSerializer({
             'user': user,
             'peer_approval_enforced': settings.PEER_APPROVAL_ENFORCED,
+            'logout_url': settings.OIDC_LOGOUT_URL or '/control/logout',
```
Comment (rehandalal) [inline recipe-server/normandy/base/api/views.py]: We should probably use `reverse` to get the fallback URL so we don't have to keep this updated.

File: recipe-server/client/control_new/components/common/CurrentUserDetails.js (hunk 2 above)
Comment (mythmon) [inline recipe-server/client/control_new/components/common/CurrentUserDetails.js]: This sounds like something that should be a lint so it always gets enforced.

**Pull request merged**